crates/joker/src/error.rs

```diff
@@ -20,51 +21,45 @@ pub enum Error {
     ReservedWordWithEscapes(Reserved)
 }
 
+impl Error {
+    fn as_str(&self) -> &'static str {
```

Comment:
My bad, didn't read my implementation closely enough. I had some different code here earlier that has since changed; it's correct to consolidate these two methods.